ios/Base58.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
```

Comment:
Is this routine borrowed from somewhere? If so, cite the reference or import the library. If this is written from scratch it needs unit tests.

Reply:
I could not figure out how to integrate native tests into the React Native workflow, but I did use them when I implemented the routines: https://github.com/swansontec/native-base58